docker/incremental_load.sh.tpl

```diff
@@ -72,6 +72,17 @@ function tag_layer() {
   fi
 }
 
+function read_variables() {
+  local file=${RUNFILES}/$1
+  local new_file=$(mktemp)
+
+  sed -E "s/^([^ ]+) (.*)\$/export \\1='\\2'/g" < ${file} > ${new_file}
```

Comment:
Not sure either of these is better than adding a comment, we'd be trading "tricky sed" for "tricky bash".  Your suggestion is also buggy, since it doesn't quote the RHS. :)

I'm adding a descriptive comment.